rustfmt.toml

```diff
@@ -7,7 +7,7 @@ fn_brace_style = "PreferSameLine"
 fn_return_indent = "WithArgs"
 fn_args_paren_newline = true
 fn_args_density = "Compressed"
-fn_args_layout = "Visual"
+fn_args_layout = "Vertical"
```

Comment:
Changing this has caused very many changes which are just formatting, making this hard to review. I would prefer you left the formatting settings as they were. If you must change the formatting, let's revert completely to the Rust defaults, and put the formatting changes in a separate commit (as you did with the Rust 2018 changes). Thanks.

Reply:
done!

Reply:
For context, this (and many of the other options) don't exist anymore in rustfmt (1.4.9-stable (33e3667 2019-10-07)), so I've just removed them all and re-formatted the project. 

```
Warning: Unknown configuration option `closure_indent_style`
Warning: Unknown configuration option `density`
Warning: Unknown configuration option `enum_trailing_comma`
Warning: Unknown configuration option `expr_indent_style`
Warning: Unknown configuration option `fn_arg_indent`
Warning: Unknown configuration option `fn_args_density`
Warning: Unknown configuration option `fn_args_paren_newline`
Warning: Unknown configuration option `fn_brace_style`
Warning: Unknown configuration option `fn_empty_single_line`
Warning: Unknown configuration option `fn_return_indent`
Warning: Unknown configuration option `generics_indent`
Warning: Unknown configuration option `ideal_width`
Warning: Unknown configuration option `leeway`
Warning: Unknown configuration option `reorder_imported_names`
Warning: Unknown configuration option `rewrite_if_else`
Warning: Unknown configuration option `single_line_if_else`
Warning: Unknown configuration option `struct_lit_style`
Warning: Unknown configuration option `struct_lit_trailing_comma`
Warning: Unknown configuration option `struct_trailing_comma`
Warning: Unknown configuration option `where_density`
Warning: Unknown configuration option `where_indent`
Warning: Unknown configuration option `where_layout`
Warning: Unknown configuration option `where_pred_indent`
Warning: can't set `format_strings = true`, unstable features are only available in nightly channel.
Warning: can't set `fn_single_line = true`, unstable features are only available in nightly channel.
Warning: can't set `report_todo = Always`, unstable features are only available in nightly channel.
Warning: can't set `report_fixme = Never`, unstable features are only available in nightly channel.
```